src/module.cpp

```diff
@@ -128,3 +129,33 @@ void Module::VerifyFunction() {
 		}
 	}
 }
+
+const std::string Module::MapSpecie(const std::pair<std::string, int> &specie) {
```

Comment:
This function should probably be split up into two. As I see it, it maps the specie name with the module name, and adds the concentration number to the string.

And if you find yourself describing a function using the word "and", that means you should have two functions. 

An added benefit of splitting it up would be that you don't have to do the `if (specie.second != 1)` logic twice.